src/sidebar/components/user-menu.js

```diff
@@ -18,11 +18,9 @@ import SvgIcon from '../../shared/components/svg-icon';
 
 /**
  * @typedef AuthState
- * @prop {'logged-in'|'logged-out'} status
```

Comment:
What happened to status & provider?

Reply:
The code in this component doesn't use those props - it is fine if the type that is passed in is a superset of `AuthState` which does have them.